.github/workflows/deploy_to_production.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy to Production
+
+on:
+  push:
+    branches:
+      - production
+    paths-ignore:
+      - 'documentation/**'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: setup-inputs
+        run: |
+          INPUTS='${{ format('{{"environment": "production", "ref": "{0}"}}', github.sha) }}'
+          echo "INPUTS=${INPUTS}" >> $GITHUB_ENV
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: trigger-deploy
+        uses: benc-uk/workflow-dispatch@v1.1
+        with:
+          workflow: Deploy to Environment # Workflow name, in deploy.yml
+          token: ${{ secrets.PERSONAL_TOKEN }}
```

Comment:
Curious what this is for.

Reply:
Same as the deploy to every other environment. Used for permissions to trigger a workflow: [docs](https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token:~:text=If%20you%20need%20a%20token%20that%20requires%20permissions%20that%20aren't%20available)

Reply:
This is so I get an email notification when you open a production release at 2am on a Saturday